scripts/run_backend_tests.py

```diff
@@ -347,11 +352,17 @@ def main(args=None):
             'Checking whether coverage is installed in %s'
             % common.OPPIA_TOOLS_DIR)
         if not os.path.exists(
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4')):
-            python_utils.PRINT('Installing coverage')
-            install_third_party_libs.pip_install(
-                'coverage', '4.5.4',
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4'))
+                os.path.join(
+                    common.OPPIA_TOOLS_DIR,
+                    'coverage-%s' % common.COVERAGE_VERSION)):
+            raise Exception('Coverage is not installed, please run the start '
+                            'script.')
+
+        pythonpath = [COVERAGE_DIR]
```

Comment:
Usually "pythonpath" is a string. Perhaps call this pythonpath_components?

Reply:
Done!